pkg/operator/controllers/dnsmasq/machineconfig_controller.go

```diff
@@ -16,12 +16,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	"github.com/Azure/ARO-RP/pkg/operator"
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
 	"github.com/Azure/ARO-RP/pkg/operator/controllers/base"
 	"github.com/Azure/ARO-RP/pkg/util/dynamichelper"
 )
 
 const (
-	MachineConfigControllerName = "DnsmasqMachineConfig"
+	machineConfigControllerName = "DnsmasqMachineConfig"
```

Comment:
nit: super small thing, and may be 100% personal preference, but I find it's easier to separate entirely mechanical changes like changing all the names to be private into a separate commit - it keeps the core logic changes that need good scrutiny in a small change-set that's easy to review